### DISCUSSION TITLE
Setup Cloudfront url for AWS S3 provider ?What about new version of strapi. Because settings button is hidden

### DISCUSSION THREAD

**Discussion by devhackery** [strapi/strapi#6622] [Legacy Topics]
How to Setup Cloudfront url for AWS S3 provider
@rmaroun What about new version of strapi. Because settings button is hidden

_Originally posted by @B1Z1 in https://github.com/strapi/strapi/issues/3218#issuecomment-616130672_

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
